docs/parameters.yaml

```diff
@@ -655,6 +655,43 @@ type: object
 default: none
 components: ["nsregistry"]
 ---
+name: Registry.InstitutionsUrl
```

Comment:
Has this been checked with @bbockelm? Last I heard is he wanted institutions to be something set via the web interface, but my information could be out of date.

Reply:
Sorry I think I based this PR off from my earlier PR that's already merged in main, so there's extra commit diff. But yes, that InstitutionUrl is a requested feature from Brian.